packages/shopify-codemod/transforms/remove-empty-returns.js

```diff
@@ -0,0 +1,35 @@
+export default function removeEmptyReturns({source}, {jscodeshift: j}, {printOptions = {}}) {
+  const sourceAST = j(source);
+
+  sourceAST
+  .find(j.FunctionDeclaration, {
+    body: {
+      body: (lines) => j.ReturnStatement.check(lines[lines.length - 1]),
```

Comment:
Won't this kill any trailing `return` statements, even if they have an actual argument?